src/components/Redirect/RedirectFooter.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link';
+
```

Comment:
why a blank line?

Reply:
but there should be no blank line in middle of imports statement

Reply:
Yes, it separates the import based on the imported categories like external library, components, styles, etc.